build/azure-pipelines/common/sql-computeNodeModulesCacheKey.js

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
So currently we don't have these js files checked in for this folder for any of the other scripts - VS Code used to do this but it looks like they changed that a couple months back to improve compile times : 

https://github.com/microsoft/vscode/commit/d1fff1a08360f0d35f124c990dc65a6bccd748d2

But given that we don't do this for anything else currently I would say let's keep things consistent and not check in this one either since it's annoying to have to remember to update the compiled JS files for some of the scripts but not others.

Reply:
Synced offline - doing this would require some additional work to add in steps for restoring packages and building the folder (this one is special since it has to run before the node_modules restore, so can't rely on that). So for now we'll keep it like this with the checked in JS and Aasim will be following up with changes to consolidate how we're handling the files here.